README.md

```diff
@@ -1 +1,106 @@
-# AwesomeBookEs6
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+<img src="./microverse1.png" alt="logo" width="140"  height="auto" />
+<br/>
+
+<h3><b>Microverse README Template</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+  - [🚀 Live Demo](https://pierrebarba.github.io/finishProjectCapton/index.html)
+- [💻 Getting Started](https://pierrebarba.github.io/finishProjectCapton/index.html)
+  - [Setup](setup)
+  - [Prerequisites](https://pierrebarba.github.io/finishProjectCapton/index.html)
+  - [Install](install)
+  - [Usage](html,javascript,css)
+- [👥 Authors](https://github.com/PierreBarba)
+- [🔭 Future Features](https://pierrebarba.github.io/finishProjectCapton/index.html)
+- [🤝 Contributing](https://github.com/cilfonegabriel)
+- [⭐️ Show your support](support)
+- [🙏 Acknowledgements](https://github.com/elafildecolor,https://github.com/luigirazum)
+- [📝 License](MIT)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capton Project] <a name="Capton project First Module"></a>
```

Comment:
- [ ] Kindly edit the name of your project I the README file.

Reply:
Hello, thank you very much for the review. I appreciate your attention, to the project, and thanks for your time, I have reviewed my code with my morning group and I also tried it on different PCs to see the error since that part works perfectly fine, so I will modify the options and request a review again, thank you very much again I have reviewed it to be sure and now it is complete 😄